tests/bsim/bluetooth/mesh/prj.conf

```diff
@@ -74,5 +74,5 @@ CONFIG_BT_TESTING=y
 # Needed for RPR tests due to huge amount of retransmitted messages
 CONFIG_BT_MESH_MSG_CACHE_SIZE=64
 
-# Enable tinycrypt as a crypto backend
-CONFIG_BT_MESH_USES_TINYCRYPT=y
+# Increase the number of key slots in PSA Crypto core
+CONFIG_MBEDTLS_PSA_KEY_SLOT_COUNT=64
```

Comment:
Side note: We will have to write recommendation on how to compute this number. When people start changing Netkey/Appkey indices this number will change, and unfortunately Kconfig parser does not do mathematical operations.

Reply:
Unfortunately, static key slot allocation is available only here. Our Zephyr fork doesn't support it at the moment since different mbedTLS backend. They are allocated from heap.